README.md

```diff
@@ -81,7 +81,7 @@ The map of containers consists of the name of the container mapped to the contai
 	* `device` (array) Add host devices.
 	* `dns` (array)
 	* `entrypoint` (string)
-	* `env` (array)
+	* `env` (array/mapping) `env` can be declared as a mapping where each `key`-`value` will be translated to a string `"key=value"`.
```

Comment:
I think it should be made more clear that the type of `env` is _either_ an array of strings or a mapping of strings - and that the example given should be declined in both versions since none is "preferred".